apps/gallery/stories/composites/wui-transaction-list-item.stories.ts

```diff
@@ -0,0 +1,59 @@
+import type { Meta } from '@storybook/web-components'
+import '@web3modal/ui/src/composites/wui-transaction-list-item'
+import type { WuiTransactionListItem } from '@web3modal/ui/src/composites/wui-transaction-list-item'
+import { html } from 'lit'
+import '../../components/gallery-container'
+import {
+  transactionDirectionOptions,
+  transactionStatusOptions,
+  transactionTypeOptions
+} from '../../utils/PresetUtils'
+
+type Component = Meta<WuiTransactionListItem>
+
+export default {
+  title: 'Composites/wui-transaction-list-item',
+  args: {
+    type: 'approve',
+    imageURL:
+      'https://uploads-ssl.webflow.com/61fe6bb74158b468a1112105/61fe8900148b6e6ee7c2c4e4_Profile%20Pic_Smoker-p-500.jpeg',
```

Comment:
Can you add this imageURL to `PresetUtils`, so this file stays more clean?

Reply:
Definitely